virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -90,6 +90,10 @@ Overcommitting memory on a highly utilized system can decrease workload performa
 //CNV-40005 Application-aware resource quota
 * You can now use the xref:../../virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.adoc#virt-understanding-aaq-operator[Application-Aware Quota (AAQ) Operator] to customize and manage resource quotas for individual components in an {product-title} cluster. The AAQ Operator provides the `ApplicationAwareResourceQuota` and `ApplicationAwareClusterResourceQuota` custom resource definitions (CRDs) that can be used to allocate resources without interfering with cluster-level activities such as upgrades and node maintenance.
 
+//CNV-31342 Upgraded environments automatically migrate instancetype ControllerRevisions
+
+* {VirtProductName} now automatically updates `instancetype.kubevirt.io` objects saved in the `ControllerRevisions` settings to the latest version of the object. As a result, you no longer need to manually update `ControllerRevisions` if you require using updated supported versions of `instancetype.kubevirt.io` objects.
```

Comment:
```suggestion
* {VirtProductName} now automatically updates `instancetype.kubevirt.io` objects stored in `ControllerRevisions` that are referenced by `VirtualMachines` to the latest API version of the object. The latest version of the API currently being `instancetype.kubevirt.io/v1beta1`.
```

Reply:
@lyarwood , thank you for the suggested rewrites. Nevertheless, I'd prefer not to explicitly name the currently latest version, because that will most likely change soon, which in turn will mean that the RN text will become inaccurate. Also, you're right that the second sentence more or less reiterates the same information, but the point is to make the UX impact as explicit as possible.

With that in mind, how about the following description as a compromise?

>  {VirtProductName} now automatically updates `instancetype.kubevirt.io` objects stored in `ControllerRevisions` that are referenced by `VirtualMachine` resources to the latest API version of the objects. As a result, you no longer need to manually update these objects.